src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3883,12 +3894,40 @@ private VisitArgumentResult VisitArgumentEvaluate(BoundExpression argument, RefK
             Debug.Assert(!this.IsConditionalState);
         }
 
-        private void CheckDisallowedNullAssignment(TypeWithState state, FlowAnalysisAnnotations annotations, Location location)
+        private void CheckDisallowedNullAssignment(TypeWithState state, FlowAnalysisAnnotations annotations, Location location, BoundExpression boundValueOpt = null)
```

Comment:
Consider naming `CheckDisallowNullAssignment` to more strongly denote that this is about DisallowNullAttribute

Reply:
I think that "check disallowed null assignment" rings better as an action. I'll keep as-is

---
In reply to: [368161189](https://github.com/dotnet/roslyn/pull/40789#discussion_r368161189) [](ancestors = 368161189)